package.json

```diff
@@ -21,104 +21,88 @@
-    "babel-plugin-transform-es2015-shorthand-properties": "^6.22.0",
-    "babel-plugin-transform-es2015-spread": "^6.22.0",
-    "babel-plugin-transform-es2015-template-literals": "^6.22.0",
-    "broccoli-asset-rev": "^2.4.5",
-    "broccoli-babel-transpiler": "^6.4.3",
-    "broccoli-concat": "^3.2.2",
-    "broccoli-stew": "^1.4.2",
-    "broccoli-string-replace": "^0.1.1",
-    "broccoli-test-helper": "^1.2.0",
-    "broccoli-uglify-sourcemap": "^1.0.1",
+    "@ember-decorators/babel-transforms": "^2.0.2",
+    "@ember-decorators/data": "^2.3.1",
+    "babel-eslint": "^8.2.6",
+    "broccoli-asset-rev": "^2.7.0",
+    "broccoli-babel-transpiler": "^6.5.0",
+    "broccoli-concat": "^3.5.1",
+    "broccoli-stew": "^2.0.0",
+    "broccoli-string-replace": "^0.1.2",
+    "broccoli-test-helper": "^1.4.0",
+    "broccoli-uglify-sourcemap": "^2.2.0",
     "co": "^4.6.0",
-    "common-tags": "^1.4.0",
-    "ember-cli": "^3.1.4",
-    "ember-cli-app-version": "^3.0.0",
+    "common-tags": "^1.8.0",
+    "ember-cli": "^3.3.0",
+    "ember-cli-app-version": "^3.2.0",
     "ember-cli-blueprint-test-helpers": "^0.18.3",
-    "ember-cli-dependency-checker": "^2.1.0",
-    "ember-cli-eslint": "^4.2.3",
-    "ember-cli-htmlbars": "^2.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
-    "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-internal-test-helpers": "^0.8.1",
+    "ember-cli-dependency-checker": "^3.0.0",
+    "ember-cli-htmlbars": "^3.0.0",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-internal-test-helpers": "^0.9.1",
     "ember-cli-pretender": "^1.0.1",
     "ember-cli-release": "^0.2.9",
-    "ember-cli-shims": "^1.0.2",
-    "ember-cli-sri": "^2.1.0",
+    "ember-cli-shims": "^1.2.0",
+    "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "^2.2.0",
-    "ember-cli-uglify": "2.0.0-beta.1",
+    "ember-cli-uglify": "2.1.0",
     "ember-cli-yuidoc": "^0.8.8",
-    "ember-decorators": "^2.1.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-decorators": "^2.3.1",
+    "ember-disable-prototype-extensions": "^1.1.3",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-publisher": "0.0.7",
-    "ember-qunit": "^3.4.0",
+    "ember-qunit": "^3.4.1",
     "ember-qunit-assert-helpers": "^0.2.1",
-    "ember-resolver": "^4.1.0",
+    "ember-resolver": "^5.0.1",
     "ember-source": "~3.0.0",
-    "ember-source-channel-url": "^1.0.1",
-    "ember-try": "^0.2.23",
-    "ember-watson": "^0.7.0",
+    "ember-source-channel-url": "^1.1.0",
+    "ember-try": "^1.0.0-beta.3",
+    "eslint": "^5.3.0",
     "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-prettier": "2.6.0",
-    "fs-extra": "^6.0.1",
+    "eslint-plugin-node": "^7.0.1",
```

Comment:
Major version bump due to dropping Node 4 support (also adds more strict rules about things available in Node 10)